app/minutes/controllers.py

```diff
@@ -240,15 +239,14 @@ def edit_minute(user, user_data):
 
     membership = committee.members.filter_by(member= user).first()
 
-    if (user.id != committee.head and user.is_admin == False and
-        (membership is None or membership.role != Roles.MinuteTaker)):
+    if membership is None and not user.is_admin:
         emit('edit_minute', Response.PermError)
         return
     
-    if (user.id != committee.head and 
-        "private" in user_data and not user_data["private"]):
-        emit('create_minute', Response.PermError)
-        return
+    if "private" in user_data and not user_data["private"]:
```

Comment:
Probably just a nitpick, but I noticed earlier you were using the get method on the object instead of the direct array accessor. In this instance it would simplify the logic to do `user_data.get("private", false)` instead of checking for the existence of "private", and then comparing its truthiness.